From ea4d78ac3fc01f33bc2c0f3fb4a44a4103196da5 Mon Sep 17 00:00:00 2001
From: Antonio Rojas <arojas@archlinux.org>
Date: Fri, 23 Feb 2018 21:51:45 +0100
Subject: Fix build with VLC<3

libvlc_title_description_t and libvlc_chapter_description_t are not defined in vlc < 3

Differential Revision: https://phabricator.kde.org/D10775
---
 src/mediaplayer.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mediaplayer.h b/src/mediaplayer.h
index 9d413d4..cd847db 100644
--- a/src/mediaplayer.h
+++ b/src/mediaplayer.h
@@ -62,11 +62,13 @@ private:
     unsigned int m_size;
 };
 
+#if (LIBVLC_VERSION_INT >= LIBVLC_VERSION(3, 0, 0, 0))
 typedef Descriptions<libvlc_title_description_t> TitleDescriptions;
 typedef QSharedPointer<const TitleDescriptions> SharedTitleDescriptions;
 
 typedef Descriptions<libvlc_chapter_description_t> ChapterDescriptions;
 typedef QSharedPointer<ChapterDescriptions> SharedChapterDescriptions;
+#endif
 
 class MediaPlayer : public QObject
 {
-- 
cgit v0.11.2


